web/frontend/src/components/slam/index.svelte

```diff
@@ -93,6 +94,38 @@ const startDurationTimer = (start: number) => {
   }, 400);
 };
 
+const setMappingMode = async () => {
+  console.log("in setMappingMode, slamModel: ", overrides?.slamModel)
+  console.log("in setMappingMode, overrides.mode: ", overrides?.mappingDetails.mode)
+  console.log("in setMappingMode, mappingMode: ", mappingMode)
+  // modes: 'localize' | 'create' | 'update' | 'undefined'
```

Comment:
If `mappingMode` will only ever be this union of strings, can we set that as the type above?

```ts
let mappingMode = 'localize' | 'create' | 'update' | 'undefined' = 'undefined'
```

Reply:
Suggestion revised, see comment below!